generators/app/templates/ext-ts/renderer.tsx

```diff
@@ -1,16 +1,263 @@
-import { LensRendererExtension } from "@k8slens/extensions";
-import { ExampleIcon, ExamplePage } from "./page"
+import { LensRendererExtension, Interface, Component } from "@k8slens/extensions";
 import React from "react"
 
-export default class ExampleExtension extends LensRendererExtension {
-    clusterPages = [
-        {
-            path: "/extension-example",
-            title: "Hello World",
-            components: {
-                Page: () => <ExamplePage extension={this} />,
-                MenuIcon: ExampleIcon,
-            }
-        }
-    ]
+const { Icon, PageLayout } = Component;
+
+/**
+ * 
+ * RendererExtension which extends LensRendererExtension runs in Lens' 'renderer' process (NOT 'main' process)
+ * main vs renderer <https://www.electronjs.org/docs/tutorial/quick-start#main-and-renderer-processes>
+ * 
+ * LensRendererExtension is the interface to Lens' renderer process. Its api allows you to access, configure, 
+ * and customize Lens data add custom Lens UI elements, and generally run custom code in Lens' renderer process.
+ * 
+ * The custom Lens UI elements that can be added include global pages, cluster pages, 
+ * cluster features, app preferences, status bar items... See details:
+ * <https://docs.k8slens.dev/master/extensions/capabilities/common-capabilities/#renderer-extension>
+ *
+ * LensRendererExtension API doc <https://docs.k8slens.dev/master/extensions/api/classes/lensrendererextension/>
+ *
+ * To see console statements in 'renderer' process, go to the console tab in DevTools in Lens
+ * View > Toggle Developer Tools > Console.
+ * 
+ * @export
+ * @class RendererExtension
+ * @extends {LensRendererExtension}
+ */
+export default class RendererExtension extends LensRendererExtension {
+
+  /**
+   *  `globalPages` allows you register custom global page.
+   * 
+   *  The global page is a full-screen page that hides all other content from a window.
+   *
+   *  ```
+   *            Lens
+   *   +-----------------------+
+   *   |*|                     |
```

Comment:
Does global page also hide left menu?

Reply:
Yeah, You are right

Reply:
fixed in https://github.com/lensapp/generator-lens-ext/pull/11/commits/9d3e5011b0c9c38ee61a6ab6eeed0cfe4a65cb18